glustercli/cmd/georep.go

```diff
@@ -45,18 +46,14 @@ Rerun the Session Create command with --force once the issues related to Remote
 )
 
 var (
-	flagGeorepCmdForce            bool
-	flagGeorepShowAllConfig       bool
-	flagGeorepRemoteGlusterdHTTPS bool
-	flagGeorepRemoteGlusterdHost  string
-	flagGeorepRemoteGlusterdPort  int
+	flagGeorepCmdForce        bool
+	flagGeorepShowAllConfig   bool
+	flagGeorepRemoteEndpoints string
 )
 
 func init() {
 	// Geo-rep Create
-	georepCreateCmd.Flags().BoolVarP(&flagGeorepRemoteGlusterdHTTPS, "remote-glusterd-https", "", false, "Remote Glusterd HTTPS")
-	georepCreateCmd.Flags().StringVarP(&flagGeorepRemoteGlusterdHost, "remote-glusterd-host", "", "", "Remote Glusterd Host")
-	georepCreateCmd.Flags().IntVarP(&flagGeorepRemoteGlusterdPort, "remote-glusterd-port", "", 24007, "Remote Glusterd Port")
+	georepCreateCmd.Flags().StringVar(&flagGeorepRemoteEndpoints, "remote-endpoints", "http://127.0.0.1:24007", "remote glusterd2 endpoints")
```

Comment:
In other place, this is replaced with the hostname of the remote node specified, so I think not a problem. But keeping this as empty string will help to avoid confusion.

Reply:
done